AdaptiveJetStream/jetstream/src/main/java/com/google/jetstream/presentation/screens/moviedetails/components/MovieDetails.kt

```diff
@@ -77,16 +77,15 @@ fun MovieDetails(
 
     Box(
         modifier = Modifier
-            .fillMaxWidth()
-            .height(432.dp)
-            .bringIntoViewRequester(bringIntoViewRequester)
+            .wrapContentSize()
```

Comment:
Would you please confirm if the `Box` is brought into viewport when the `WatchTrailerButton` has keyboard focus.

Reply:
Would you also check if the details screen is properly rendered on  TV?